src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -187,6 +201,38 @@ export default Vue.extend({
 
     displayVideoPlayButton: function() {
       return this.$store.getters.getDisplayVideoPlayButton
+    },
+    formated_stats: function() {
+      let resolution = ''
+      let dropFrame = ''
+      if (this.stats.playerResolution != null) {
+        resolution = `(${this.stats.playerResolution.height}X${this.stats.playerResolution.width})`
+      }
+      if (this.stats.frameInfo != null) {
+        dropFrame = `${this.stats.frameInfo.droppedVideoFrames} out of ${this.stats.frameInfo.totalVideoFrames}`
+      }
+      const stats = [
+        ['video id', this.stats.videoId],
+        ['player resolution', resolution],
+        ['volume', this.stats.volume.toFixed(2)],
+        ['fps', this.stats.fps],
+        ['frame drop', dropFrame],
+        ['network state', this.stats.networkState],
+        ['bandwidth', `${(this.stats.bandwidth / 1000).toFixed(2)} kbps`],
+        ['% buffered', this.stats.bufferPercent.toFixed(2)]
+      ]
+
+      let formatedStats = '<ul style="list-style-type: none;text-align:left">'
+      for (const stat of stats) {
+        formatedStats += `<li >${stat[0]}: ${stat[1]}</li>`
+      }
+      formatedStats += '</ul>'
+      return formatedStats.substring(0, formatedStats.length - 1)
```

Comment:
```suggestion
      return formatedStats.substring(0, formatedStats.length)
```
-1 shouldn't be here